analyzers/rspec/cs/S1192_c#.html

```diff
@@ -32,9 +32,7 @@ <h2>Compliant Solution</h2>
 </pre>
 <h2>Exceptions</h2>
 <p>The following are ignored:</p>
-<ul>
-  <li> literals with fewer than 5 characters </li>
-  <li> literals matching one of the parameter names </li>
-  <li> literals used in attributes </li>
-</ul>
+<p> * literals with fewer than 5 characters</p>
```

Comment:
this shouldn't be included

this happens when there is a space before the `*` in JIRA (yes 🤦‍♂️ , we need to manually remove the spaces )